### ISSUE TITLE
Header pattern: Moving keyboard focus from the last link in the megamenu to body content changes the colour of the header to black.

### ISSUE THREAD

**Issue by jkmnch** [wmcadigital/wmn-design-system#464]
**Describe the bug**
Moving keyboard focus from the last link in the megamenu to the updates button in the footer changes the colour of the header to black, and the grey overlay goes over the megamenu dropdown, which doesn't disappear.

**Expected behaviour**
The keyboard focus should move to the next link in the body content and the megamenu should stay purple and dropdown links should dissapear.

**Screenshots**
![ezgif com-gif-maker](https://user-images.githubusercontent.com/62507489/96847231-d6dee780-144a-11eb-90a5-1f107c79fa34.gif)

Sorry, it's a low-quality GIF, I can't upload videos to Github. If you want the full video, let me know on Teams.

**Desktop (please complete the following information):**
 - OS: macOS 10.15.7
 - Browser: Safari
 - Version: 14.0

**Comment by houbly** [comment]
This should be the same issue as #462 so I'll address them together
